From a89a48ddd94d5392d244887959d735dc0f0623be Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@nxp.com>
Date: Thu, 19 Oct 2017 19:00:02 +0800
Subject: [PATCH 2660/5242] MA-10480 Make CONFIG_I2C_IMX to depend on
 CONFIG_ARCH_MXC_ARM64

commit  76435c5225314ea7fbc0e3f917580ad637c6ef60 from
https://source.codeaurora.org/external/imx/linux-imx.git

imx8mscale evk uses the i2c imx driver to control the pfuze driver
otherwise pfuze driver wont be probed with I2C_IMX

Change-Id: Iaeacde58a4cbe34a3d18cb16814d2334c74c2b79
(cherry-picked from commit ad7200824fa740a1fe9d418d3f949ff97b083bdf)
Signed-off-by: guoyin.chen <guoyin.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/i2c/busses/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/i2c/busses/Kconfig b/drivers/i2c/busses/Kconfig
index 879187c..805a360 100644
--- a/drivers/i2c/busses/Kconfig
+++ b/drivers/i2c/busses/Kconfig
@@ -628,7 +628,7 @@ config I2C_IMG
 
 config I2C_IMX
 	tristate "IMX I2C interface"
-	depends on ARCH_MXC || ARCH_LAYERSCAPE || COLDFIRE
+	depends on ARCH_MXC || ARCH_LAYERSCAPE || COLDFIRE || ARCH_MXC_ARM64
 	help
 	  Say Y here if you want to use the IIC bus controller on
 	  the Freescale i.MX/MXC, Layerscape or ColdFire processors.
-- 
1.7.9.5

